app/src/main/java/de/tum/in/tumcampusapp/managers/EduroamManager.java

```diff
@@ -118,8 +118,7 @@ public boolean configureEduroam(String lrzId, String networkPass) {
             return false;
         }
 
-        //Save, enable and exit
-        wifiManager.saveConfiguration();
```

Comment:
Nope. The method's deprecated now.
https://developer.android.com/reference/android/net/wifi/WifiManager.html#saveConfiguration()
> This method was deprecated in API level 26.
> There is no need to call this method - addNetwork(WifiConfiguration), updateNetwork(WifiConfiguration) and removeNetwork(int) already persist the configurations automatically.